### ISSUE TITLE
[Rule change] `prefer-extracting-callbacks` shouldn't trigger on empty callback

### ISSUE THREAD

**Issue by ookami-kb** [dart-code-checker/dart-code-metrics#458]
**What rule do you want to change?**

prefer-extracting-callbacks

**How will the change be implemented? (New option, new default behavior, etc.)?**

New behavior: if the callback is empty, it shouldn't trigger the rule. Having an empty callback doesn't make the readability worse.

**Please provide some example code that this change will affect:**

```dart
final button = TextButton(
  child: const Text('Push me'),
  onPressed: () {},
);
```

**What does the rule currently do for this code?**

Triggers.

**What will the rule do after it's changed?**

Doesn't trigger.

**Are you willing to submit a pull request to implement this change?**

Yes.

**Comment by incendial** [comment]
@ookami-kb good catch! If you want to implement it by yourself please let me know if you have any questions about the contribution process.

**Comment by dkrutskikh** [comment]
Landed in 4.4.0
